### PR TITLE
[type:Integration Test] add integrated-test of logging-rocketmq-plugin

### PR DESCRIPTION
**Pull request by erdengk**
This pr is used to test whether rocketmq is integrated into shenyu.
The test case steps are as follows:
1. Add a message to rocketmq
2. Consume this information

If the consumption is successful, it means that rocketmq is integrated into shenyu. If the consumption fails, an exception will be thrown, indicating that there is a problem with rocketmq.

### CODE REVIEW

**Comment by erdengk** [comment]
@JooKS-me  please review this pr,thanks~

File: shenyu-integrated-test/shenyu-integrated-test-http/docker-compose.yml [hunk 1]
```diff
@@ -128,6 +128,47 @@ services:
...
+    volumes:
+      - ./logs:/root/logs
+    command: sh mqnamesrv
+    networks:
+      shenyu:
+        aliases:
+          -  shenyu-rocketmq
+
+  shenyu-rmqbroker:
+    image: rocketmqinc/rocketmq:4.3.2
```
Comment (JooKS-me) [inline shenyu-integrated-test/shenyu-integrated-test-http/docker-compose.yml:151]: Why use `rocketmqinc/rocketmq`, is it different from `apache/rocketmq` ?

File: .github/workflows/integrated-test.yml [hunk 2]
```diff
@@ -20,7 +20,6 @@ on:
   push:
     branches:
       - master
-
```
Comment (JooKS-me) [inline .github/workflows/integrated-test.yml:23]: Don't delete this line

**Review by JooKS-me — CHANGES_REQUESTED** [review]

File: shenyu-integrated-test/shenyu-integrated-test-http/docker-compose.yml (hunk 1 above)
Comment (erdengk) [reply shenyu-integrated-test/shenyu-integrated-test-http/docker-compose.yml:151]: There is no difference between them, apache/rocketmq is officially released by apache. rocketmqinc/rocketmq is a private release, maybe apache/rocketmq is better here.

Comment (JooKS-me) [reply shenyu-integrated-test/shenyu-integrated-test-http/docker-compose.yml]: Yep, let's change it to apache/rocketmq.

Comment (erdengk) [reply shenyu-integrated-test/shenyu-integrated-test-http/docker-compose.yml]: ok

**Review by JooKS-me — APPROVED** [review]
Good job

**Pull request merged**